src/core/tests/Test_UbuntuProClient.py

```diff
@@ -97,6 +100,50 @@ def mock_unimport_uaclient_reboot_required_module(self):
         self.mock_unimport_module('uaclient.api.u.pro.security.status.reboot_required.v1')
 
 
+class UpdateInfo:
+    def __init__(self, package, version, provided_by, origin):
+        self.version = version
+        self.provided_by = provided_by
+        self.package = package
+        self.origin = origin
+
+
```

Comment:
nit: remove extra empty new line

Reply:
Pycharm expects two lines between classes. Hence made it that way.